theme/src/resources/templates/category_pages/liferay_dxp_cloud.hbs

```diff
@@ -20,15 +20,6 @@
 
 		<div class="col-md-8 offset-md-2" id="dxpCloudProductPage"></div>
 
-		<div class="col-md-8 offset-md-2">
-			<div class="cta landing-page-cta">
-				<h4>{{dc 'dxp_cloud-announcements_banner_description'}}</h4>
-
-				<a class="btn btn-outline-primary btn-sm"
-					href="https://discover.liferay.com/subscribe-to-liferay-dxp-cloud-updates">
-					{{dc 'homepage-announcement_subscribe_button'}}
-				</a>
-			</div>
```

Comment:
@cole-liferay I think you want to remove the closing </div> in the next line as well.